### PR TITLE
[FLOC-1253] Document how to fix bugs in releases.

### PR DESCRIPTION
**Pull request by tomprince**
https://clusterhq.atlassian.net/browse/FLOC-1253

### CODE REVIEW

**Comment by tomprince** [comment]
This currently suggest merging into all affected branches. It may make sense to just merge into all subsequent versions. If a branch isn't affected, then the merge-forward step would presumably just revert the fix. I think this will ensure that the any subsequent fixes (which are branched off the release branch) don't have to resolve those conflicts.

**Comment by tomprince** [comment]
Also, I haven't tried the steps here. I'll do something approximating them for https://clusterhq.atlassian.net/browse/FLOC-1166.

File: docs/gettinginvolved/contributing.rst [hunk 1]
```diff
@@ -1 +1,64 @@
...
+   As :ref`bugfix releases<bugfix-releases>` aren't currently being produced,
+   the following instructions are only relevant for documentation fixes.
+
+
+Occasionally, issues will be discovered that want to be fixed before the next full release.
+The following is the procedure for fixing them.
+
+#. File an issue in JIRA_. For example, "FLOC-1234: Fix a bug.".
+   If the issue affects in multiple versions (including master),
+   create an sub-task for each affected version:
```
Comment (exarkun) [inline docs/gettinginvolved/contributing.rst]: a/an

File: docs/gettinginvolved/contributing.rst [hunk 2]
```diff
@@ -1 +1,64 @@
...
+
+   .. prompt:: bash $
+
+      git checkout -b release-maintenance/flocker-1.2.3/fix-a-bug-FLOC-1235 origin/release/flocker-1.2.3
+
+#. Fix the bug.
+
+   .. prompt:: bash $
+
+      vi ...
```
Comment (exarkun) [inline docs/gettinginvolved/contributing.rst]: vi?  Not ed? :smile:

File: docs/gettinginvolved/contributing.rst (hunk 1 above)
Comment (exarkun) [inline docs/gettinginvolved/contributing.rst]: Probably "affected, supported"

**Comment by exarkun** [comment]
Thanks.  This looks good.  One further thing we probably need to decide is what "supported" means.  Right now it probably only means "the most recent release" but I guess eventually we'll expand that.

**Comment by exarkun** [comment]
Please address trivial inline comments and merge.

**Pull request merged**